src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -63,7 +63,7 @@ This instrumentation by default collects all the incoming http requests. It
 allows filtering of requests by using `Filter` function in
 `AspNetCoreInstrumentationOptions`. This defines the condition for allowable
 requests. The Filter receives the `HttpContext` of the incoming
-request, and does not instrument the request if the Filter returns false or throws
+request, and does not collect telemetry about the request if the Filter returns false or throws
```

Comment:
can you make same change in line 79 in code comment?

Reply:
Updated 👍